src/app/service/crew/create/_components/Step2.tsx

```diff
@@ -0,0 +1,105 @@
+import { useEffect } from 'react';
+import {
+  activeSelectStyles,
+  commonSelectStyles,
+  inputClass,
+  labelClass,
+} from './UI/commonClass';
+import { StepProps } from '@/types/crew/createFormSteps';
+import { twMerge } from 'tailwind-merge';
+import { useWatch } from 'react-hook-form';
+import { Checkbox } from '@/app/_components/ui/checkbox';
+
+function Step3({ formMethods, setIsStepValid }: StepProps) {
+  const { register, setValue, trigger, getValues, control } = formMethods;
+
+  const questionValue = useWatch({ control, name: 'question' });
+  const permissionValue = useWatch({ control, name: 'permission_required' });
+
+  useEffect(() => {
+    setIsStepValid(false);
+  }, []);
+
+  useEffect(() => {
+    if (getValues('permission_required') === false) {
+      setValue('question', '');
+      setIsStepValid(true);
+    } else if (getValues('permission_required') === true) {
+      setIsStepValid(false);
+      if (questionValue) {
+        setValue('question', questionValue);
+        setIsStepValid(true);
+      }
+    }
```

Comment:
`else` 문이 존재하지 않지만 `else if` 문을 설정하신 것 같아요!
`else if`를 지양하는 것이 좋다는 관점에서 가독성 증진도 이루어내기 위해 바꿔보는 건 어떨까요?

[else, else if문을 지양하는 것이 좋은 이유](https://html-jc.tistory.com/473)

```suggestion
   if (!getValues('permission_required')) {
      setValue('question', '');
      setIsStepValid(true);
  } else {
    if (questionValue) {
      setValue('question', questionValue);
      setIsStepValid(true);
    } else {
      setIsStepValid(false);
    }
  }
```

Reply:
오 감사합니다 확실히 else문 안쓰는게 더 명확해지네요!
다만 getValues의 값이 undefined인 경우와 구분하기 위해 true / false와 비교하는 로직은 유지하겠습니다!